web-components/src/components/folksnomy/folksonomy-property-products.ts

```diff
@@ -350,6 +350,8 @@ export class FolksonomyPropertyProducts extends LitElement {
 
   override async connectedCallback() {
     super.connectedCallback()
+    // Note: The propertyName is set once and cannot be changed dynamically.
+    // If you need to change it, you must reinitialize the component.
```

Comment:
I would prefer it to have it in the description of the component, because it's an information to developers using this component.